Dockerfile

```diff
@@ -11,6 +11,40 @@ RUN apt-get update && \
   apt-get -y upgrade && \
   rm -rf /var/lib/apt/lists/*
 
+
+### ---- Front-end builder image ----
+FROM node:20 as front-builder
+
+# Copy frontend app sources
+COPY ./src/frontend /builder
+
+WORKDIR /builder/apps/desk
```

Comment:
I should copy only desk

Reply:
based on @AntoLC use smth similar as https://github.com/openfun/marsha/blob/129de945a5fada632ba3faf6538a6a19525b7343/Dockerfile#L36
to cache dependencies

Reply:
Alright, I took into account your feedbacks, and slightly modified your suggestion, but I am not 100% sure if I should copy all package.json before running `yarn install`. I red again the Marsha docker file, and they seem to follow the latter approach. What do you think?

Reply:
I refacto this part, lmk if it's better!